contrib/loaders/flash/gd32v/gd32vf103.inc

```diff
@@ -0,0 +1,8 @@
+/* Autogenerated with ../../../../src/helper/bin2char.sh */
```

Comment:
Can you add the source required to build this file to OpenOCD as well?

E.g. for FESPI we have https://github.com/riscv/riscv-openocd/blob/riscv/contrib/loaders/flash/fespi/riscv32_fespi.inc and https://github.com/riscv/riscv-openocd/blob/riscv/contrib/loaders/flash/fespi/riscv64_fespi.inc, which can be recreated by running `make` in https://github.com/riscv/riscv-openocd/tree/riscv/contrib/loaders/flash/fespi.

Reply:
Yes, I think that I can upstream this into mainline OpenOCD.

I don't have the source for contrib/loaders/flash/gd32v/gd32vf103.inc as it isn't in the repository https://github.com/riscv-mcu/riscv-openocd/

../../../../src/helper/bin2char.sh simply converts a given binary file to a series of C constants.

I see two approaches that I can try:
1) I can try to contact Kongou Hikari, the committer of contrib/loaders/flash/gd32v/gd32vf103.inc in https://github.com/riscv-mcu/riscv-openocd/ and ask for a copy of the source
2) I can refactor contrib/loaders/flash/gd32v/gd32vf103.inc to RISC-V assembly code, that can be assembled to produce a binary file, that can then be converted with the bin2char.sh tool into an identical contrib/loaders/flash/gd32v/gd32vf103.inc

Reply:
I have managed to make contact with Kongou Hikari from nucleisys.com;

> This port is done by GigaDevice, I haven't the actual source code, too. But it is basically small modification based on STM32 flash controller driver.
>
>Since I'm very busy now so I haven't so much time to rebase this fork to upstream, when these projects finished, I will try to make some effort on it.
>
>**I will ask GigaDevice for source code, but I don't know whether or not they provide it to me.**

Reply:
If the original source is hard to dig up, I'm OK merging this as-is. It would be much better to add that source at some point, in case there is ever an issue that needs to be debugged.